tests/common_docker.py

```diff
@@ -101,16 +101,31 @@ def docker_compose_cmd(arg_list, use_common_files=True, env=None):
         raise Exception("failed to start docker-compose (called: %s): exit code: %d, output: %s" % (e.cmd, e.returncode, e.output))
 
 
-def stop_docker_compose():
+def stop_docker_compose(exclude=[]):
```

Comment:
I would split these into two separate functions. They are not sharing any code whatsoever.

Reply:
+1

Reply:
you mean like - `stop_docker_compose` and `stop_docker_compose_exclude`?

Reply:
Something like that.

My suggestions:
- `stop_docker_compose_full` + stop_docker_compose_partial`
- `stop_docker_compose_all` + stop_docker_compose_exclude`

I don't know, something like that